### DISCUSSION TITLE
TRRSソケットとスイッチのはんだ付けの記載について

### DISCUSSION THREAD

**Discussion by rinyacho** [Yowkees/keyball#147] [Ideas]
ビルドガイドの「3-6．TRRSソケット・タクトスイッチ・4連ピンソケット」についてです。
「部品は【表面】から挿入して、【裏面】を下写真のように半田付けしてください。」と言及してある箇所の写真を見ると、TRRSソケットとスイッチがはんだ付けされていないように見えます。
https://github.com/Yowkees/keyball/blob/main/keyball44/doc/rev1/images/kb44_044.jpg
私はこれで終盤まではんだ付けせずに進んでしまいましたが、どうなんでしょうか。

ビルドガイド自体は全体的に丁寧で分かりやすかったと思います！😁

**Comment by Yowkees** [comment]
ビルドガイド、全体的には分かりやすかったということでありがとうございます。
確かにあの写真では間違える可能性がありそうでしたので別の写真に差し替えました。
ご指摘ありがとうございました！

**Comment by rinyacho** [comment]
ご返信ありがとうございます。
もう一点、気になったところを挙げておきます。

ビルドガイドの「3-8.ProMicroの書き込みと動作確認」についてです。
https://github.com/Yowkees/keyball/blob/main/keyball44/doc/rev1/buildguide_jp.md#3-8promicro%E3%81%AE%E6%9B%B8%E3%81%8D%E8%BE%BC%E3%81%BF%E3%81%A8%E5%8B%95%E4%BD%9C%E7%A2%BA%E8%AA%8D
文中に「すべてのキーを試す必要はまだありません」と記載がありますが、以後のどのタイミングで試すべきかの記載が見つけられませんでした（見落としていたらすみません）。
このタイミングで全てのキー入力をショート確認して、入力がうまくいかない場合は、はんだをやり直す対応をした方が手戻りが少なく済むのではと個人的には感じました。少なくとも、「7-3.保護プレートの取り付け」までに試さないと、はんだがし辛くなるように思います。
ご参考までに！

**Comment by Yowkees** [comment]
ご意見ありがとうございます。
私の思いとしては、ピンセットで全キーをテストするのはとても大変なので、全キーチェックはキースイッチをすべて取り付けた後（キーキャップすら取り付けた後）で良いかと思います。
その状態ならテストは楽ですし、不具合があってもボトムプレートを取り外せば半田面にアクセスできます。いつも私はそのように組み立てています。
今後ともどうぞよろしくお願いします！

**Comment by rinyacho** [comment]
承知しました。
ご丁寧な返信、ありがとうございました！
